Makefile

```diff
@@ -53,15 +54,18 @@ onos-config-base-docker: # @HELP build onos-config base Docker image
 onos-config-docker: onos-config-base-docker # @HELP build onos-config Docker image
 	docker build . -f build/onos-config/Dockerfile \
 		--build-arg ONOS_CONFIG_BASE_VERSION=${ONOS_CONFIG_VERSION} \
-		-t onosproject/onos-config:${ONOS_CONFIG_VERSION}
+		-t ${DOCKER_REPOSITORY}onos-config:${ONOS_CONFIG_VERSION}
```

Comment:
`${DOCKER_REPOSITORY}` does not seem to be set to any default here?

Reply:
good catch, fixed.

Reply:
If I understand this change correctly, DOCKER_REPOSITORY should have a default of onosproject and this line should read ${ONOS_REPOSITORY}/onos-config

Reply:
I was starting to adopt the convention we used in Voltha, which was that each Makefile would expose a `DOCKER_REPOSITORY` environment variable, then for example a user does `DOCKER_REPOSITORY=myprivaterepo/ make images`. We did this rather than creating uniquely named environment variables for each repo. If it would be convenient to have discussion about how the variables are named we can do that next week -- I don't have a strong opinion, and just used the convention I was already used to. The default was missing from the initial patch; I've pushed an update today. There's a similar patch up for review for the config-models repo.